micro-grizzly-with-jersey/src/test/java/app/publisher/binder/direct/AsyncResource.java

```diff
@@ -0,0 +1,40 @@
+package app.publisher.binder.direct;
+
+import com.aol.micro.server.auto.discovery.Rest;
+import cyclops.async.Future;
+import cyclops.stream.ReactiveSeq;
+import cyclops.stream.Spouts;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import java.util.concurrent.Executors;
+import java.util.stream.Stream;
+
+
+@Rest
+@Path("/test")
+public class AsyncResource {
+
+    private void sleep() {
+        try {
+            Thread.sleep(10000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+    }
+
+    @GET
+    public Future<String> myEndPoint() {
+        return Future.ofSupplier(() -> {
+            sleep();
+            return "hello world!";
+        }, Executors.newFixedThreadPool(1));
+    }
+
+    @GET
+    public ReactiveSeq<String> async2() {
+        return Spouts.publishOn(Stream.of("hello"), Executors.newFixedThreadPool(1));
```

Comment:
It should be, I think only during manual testing. Will add an automated test for it.